default.nix

```diff
@@ -243,6 +245,14 @@ let
     };
 
     marlowe-playground = rec {
+      documentation-site = let
+        # TODO: the playgroundUrl needs to be set to whatever will actually be appropriate when it's bundled
```

Comment:
This TODO is not needed - it's fine. Can you also remove the corresponding one in `plutus-playground`?